src/stores/manager/service.go

```diff
@@ -144,16 +160,21 @@ func (m *BaseManager) GetKeyStore(_ context.Context, name string) (keys.Store, e
 	return nil, errors.NotFoundError(errMessage)
 }
 
-func (m *BaseManager) GetEth1Store(ctx context.Context, name string) (eth1store.Store, error) {
+func (m *BaseManager) GetEth1Store(ctx context.Context, name string, userInfo *authtypes.UserInfo) (eth1store.Store, error) {
 	m.mux.RLock()
 	defer m.mux.RUnlock()
-	return m.getEth1Store(ctx, name)
+	return m.getEth1Store(ctx, name, userInfo)
 }
 
-func (m *BaseManager) getEth1Store(_ context.Context, name string) (eth1store.Store, error) {
+func (m *BaseManager) getEth1Store(ctx context.Context, name string, userInfo *authtypes.UserInfo) (eth1store.Store, error) {
 	if storeBundle, ok := m.eth1Accounts[name]; ok {
 		if store, ok := storeBundle.store.(eth1store.Store); ok {
-			return store, nil
+			policies := m.policyManager.UserPolicies(ctx, userInfo)
+			resolvr, err := policy.NewResolver(policies)
+			if err != nil {
+				return nil, err
+			}
+			return connectors.NewEth1Connector(store, resolvr, storeBundle.logger), nil
```

Comment:
I didn't want to go further in my implementation of Resolvers since you mentioned you were going to do work on it

Reply:
lets cover this as part of the PolicyResolver ticket as it could be consider an optimization over the way we handle policy resolvers https://app.zenhub.com/workspaces/orchestrate-5ea70772b186e10067f57842/issues/consensys/quorum-key-manager/117